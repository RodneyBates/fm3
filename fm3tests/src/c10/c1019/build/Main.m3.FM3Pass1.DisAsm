 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C621,L1(16_1),[33,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[34,2])
 ItkStmtListLt(L0(16_0),[34,2])
 ItkBlockDeclListLtPatch(C22,L1(16_1),[9,2])
 ItkBlockDeclListRt(L1(16_1),[9,2])
 ItkProcWBodyLtPatch(C22,Id8"P")
 ItkProcWBodyRt(Id8"P")
 ItkDeclScopeRt(L2(16_2))
 ItkProcBodyRt([11,2])
 ItkLookupScopeRt(L2(16_2))
 ItkStmtListLtPatch(C390,L5(16_5),[24,6])
 ItkStmtListRt(L5(16_5),[24,6])
 ItkStmtListSepPatch(C548,L4(16_4),[30,4])
 ItkBecomesLtPatch(C548,[30,8])
 ItkBecomesInfixPatch(C553,[30,8])
 ItkBecomesRt([30,8])
 ItkIdRefAtom(Id7"IC",[30,11])
 ItkIdRefAtom(Id9"V",[30,6])
 ItkStmtListSepPatch(C515,L3(16_3),[29,4])
 ItkBecomesLtPatch(C515,[29,8])
 ItkBecomesInfixPatch(C520,[29,8])
 ItkBecomesRt([29,8])
 ItkIdRefAtom(Id6"IA1",[29,11])
 ItkIdRefAtom(Id9"V",[29,6])
 ItkStmtListSepPatch(C482,L2(16_2),[28,4])
 ItkBecomesLtPatch(C482,[28,8])
 ItkBecomesInfixPatch(C487,[28,8])
 ItkBecomesRt([28,8])
 ItkIdRefAtom(Id5"I1",[28,11])
 ItkIdRefAtom(Id9"V",[28,6])
 ItkStmtListSepPatch(C449,L1(16_1),[26,4])
 ItkBecomesLtPatch(C449,[26,8])
 ItkBecomesInfixPatch(C454,[26,8])
 ItkBecomesRt([26,8])
 ItkIdRefAtom(Id4"M3",[26,11])
 ItkIdRefAtom(Id9"V",[26,6])
 ItkStmtListSepPatch(C416,L0(16_0),[25,4])
 ItkBecomesLtPatch(C416,[25,8])
 ItkBecomesInfixPatch(C421,[25,8])
 ItkBecomesRt([25,8])
 ItkIdRefAtom(Id3"MV2",[25,11])
 ItkIdRefAtom(Id9"V",[25,6])
 ItkBecomesLtPatch(C390,[24,8])
 ItkBecomesInfixPatch(C395,[24,8])
 ItkBecomesRt([24,8])
 ItkIdRefAtom(Id2"MV1",[24,11])
 ItkIdRefAtom(Id9"V",[24,6])
 ItkBlockDeclListLtPatch(C72,L7(16_7),[11,4])
 ItkBlockDeclListRt(L7(16_7),[11,4])
 ItkBlockDeclListSepPatch(C335,L6(16_6),[20,2])
 ItkConstDeclLtPatch(C335,[20,10])
 ItkConstDeclColonPatch(C341,[20,13],FALSE)
 ItkConstDeclEqualsPatch(C343,[20,13])
 ItkConstDeclRt([20,10])
 ItkIntLit(16_5(5),[20,15])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id7"IC",[20,10])
 ItkBlockDeclListSepPatch(C292,L5(16_5),[19,2])
 ItkConstDeclLtPatch(C292,[19,10])
 ItkConstDeclColonPatch(C298,[19,14],FALSE)
 ItkConstDeclEqualsPatch(C300,[19,14])
 ItkConstDeclRt([19,10])
 ItkIntLit(16_4(4),[19,16])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id6"IA1",[19,10])
 ItkBlockDeclListSepPatch(C249,L4(16_4),[18,2])
 ItkConstDeclLtPatch(C249,[18,10])
 ItkConstDeclColonPatch(C255,[18,13],FALSE)
 ItkConstDeclEqualsPatch(C257,[18,13])
 ItkConstDeclRt([18,10])
 ItkIntLit(16_3(3),[18,15])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id5"I1",[18,10])
 ItkBlockDeclListSepPatch(C206,L3(16_3),[16,2])
 ItkConstDeclLtPatch(C206,[16,10])
 ItkConstDeclColonPatch(C212,[16,13],FALSE)
 ItkConstDeclEqualsPatch(C214,[16,13])
 ItkConstDeclRt([16,10])
 ItkIntLit(16_2(2),[16,15])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id4"M3",[16,10])
 ItkBlockDeclListSepPatch(C163,L2(16_2),[15,2])
 ItkConstDeclLtPatch(C163,[15,10])
 ItkConstDeclColonPatch(C169,[15,14],FALSE)
 ItkConstDeclEqualsPatch(C171,[15,14])
 ItkConstDeclRt([15,10])
 ItkIntLit(16_1(1),[15,16])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id3"MV2",[15,10])
 ItkBlockDeclListSepPatch(C120,L1(16_1),[14,2])
 ItkConstDeclLtPatch(C120,[14,10])
 ItkConstDeclColonPatch(C126,[14,14],FALSE)
 ItkConstDeclEqualsPatch(C128,[14,14])
 ItkConstDeclRt([14,10])
 ItkIntLit(16_0(0),[14,16])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id2"MV1",[14,10])
 ItkVarDeclLtPatch(C72,[11,8])
 ItkVarDeclTypePatch(C90,[11,10])
 ItkVarDeclValPatch(C96,[14,2])
 ItkVarDeclRt([11,8])
 ItkDeclValAbsent()
 ItkReservedId(Rid"INTEGER",[11,12])
 ItkVarDeclIdListLtPatch(C72,L1(16_1),[11,8])
 ItkVarDeclIdListRt(L1(16_1),[11,8])
 ItkDeclId(L11(16_b),Id9"V",[11,8])
 ItkLookupScopeLt(L2(16_2))
 ItkProcBodyLt([11,2])
 ItkSignatureProperLtPatch(C31,[9,14])
 ItkSignatureProperRt([9,14])
 ItkRaisesSetRt(L0(16_0),[11,2])
 ItkRaisesSetLt(L0(16_0),[11,2])
 ItkResultTypeAbsent([11,2])
 ItkFormalsListLtPatch(C31,L0(16_0),[9,14])
 ItkFormalsListRt(L0(16_0),[9,14])
 ItkDeclScopeLt(L2(16_2))
 ItkDeclId(L19(16_13),Id8"P",[9,12])
 ItkLookupScopeLt(L1(16_1))
 ItkImportAS(L3(16_3),Id6"IA1",[6,15])
 ItkImportAS(L3(16_3),Id5"I1",[5,9])
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
