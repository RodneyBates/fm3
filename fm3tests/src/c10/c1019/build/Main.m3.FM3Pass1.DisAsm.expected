 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C612,L1(16_1),[33,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[34,2])
 ItkStmtListLt(L0(16_0),[34,2])
 ItkOpenDeclListLtPatch(C10,L1(16_1),[9,2])
 ItkOpenDeclListRt(L1(16_1),[9,2])
 ItkProcDefLtPatch(C10,Id5"P",[9,2])
 ItkProcDefRt(Id5"P",[31,8])
 ItkOpenScopeRt(L2(16_2))
 ItkDeclScopeRt(L3(16_3))
 ItkOpenScopeRt(L3(16_3))
 ItkStmtListLtPatch(C388,L5(16_5),[24,6])
 ItkStmtListRt(L5(16_5),[24,6])
 ItkBecomesLtPatch(C543,[30,8])
 ItkBecomesInfixPatch(C548,[30,8])
 ItkBecomesRt([30,8])
 ItkIdRefAtom(Id4"IC",[30,11])
 ItkIdRefAtom(Id6"V",[30,6])
 ItkStmtListSep(L4(16_4),[30,4])
 ItkBecomesLtPatch(C512,[29,8])
 ItkBecomesInfixPatch(C517,[29,8])
 ItkBecomesRt([29,8])
 ItkIdRefAtom(Id3"IA1",[29,11])
 ItkIdRefAtom(Id6"V",[29,6])
 ItkStmtListSep(L3(16_3),[29,4])
 ItkBecomesLtPatch(C481,[28,8])
 ItkBecomesInfixPatch(C486,[28,8])
 ItkBecomesRt([28,8])
 ItkIdRefAtom(Id2"I1",[28,11])
 ItkIdRefAtom(Id6"V",[28,6])
 ItkStmtListSep(L2(16_2),[28,4])
 ItkBecomesLtPatch(C450,[26,8])
 ItkBecomesInfixPatch(C455,[26,8])
 ItkBecomesRt([26,8])
 ItkIdRefAtom(Id9"M3",[26,11])
 ItkIdRefAtom(Id6"V",[26,6])
 ItkStmtListSep(L1(16_1),[26,4])
 ItkBecomesLtPatch(C419,[25,8])
 ItkBecomesInfixPatch(C424,[25,8])
 ItkBecomesRt([25,8])
 ItkIdRefAtom(Id8"MV2",[25,11])
 ItkIdRefAtom(Id6"V",[25,6])
 ItkStmtListSep(L0(16_0),[25,4])
 ItkBecomesLtPatch(C388,[24,8])
 ItkBecomesInfixPatch(C393,[24,8])
 ItkBecomesRt([24,8])
 ItkIdRefAtom(Id7"MV1",[24,11])
 ItkIdRefAtom(Id6"V",[24,6])
 ItkOpenDeclListLtPatch(C78,L4(16_4),[11,4])
 ItkOpenDeclListRt(L4(16_4),[11,4])
 ItkSkipLtPatch(C349,L3(16_3))
 ItkSkipRt(L3(16_3))
 ItkLiteral(96 StkIntLit,(16_5),[20,15])
 ItkConstDeclValue([20,13])
 ItkDeclTypeAbsent([20,13])
 ItkConstDeclType([20,13])
 ItkOpenDeclListSep(L4(16_4),[11,4])
 ItkSkipLtPatch(C317,L2(16_2))
 ItkSkipRt(L2(16_2))
 ItkLiteral(96 StkIntLit,(16_4),[19,16])
 ItkConstDeclValue([19,14])
 ItkDeclTypeAbsent([19,14])
 ItkConstDeclType([19,14])
 ItkOpenDeclListSep(L4(16_4),[11,4])
 ItkSkipLtPatch(C285,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkLiteral(96 StkIntLit,(16_3),[18,15])
 ItkConstDeclValue([18,13])
 ItkDeclTypeAbsent([18,13])
 ItkConstDeclType([18,13])
 ItkOpenDeclListSep(L4(16_4),[11,4])
 ItkConstDeclLtPatch(C236,[16,10])
 ItkConstDeclRt([16,10])
 ItkLiteral(96 StkIntLit,(16_2),[16,15])
 ItkConstDeclValue([16,13])
 ItkDeclTypeAbsent([16,13])
 ItkConstDeclType([16,13])
 ItkDeclId(L9(16_9),Id9"M3",L-9223372036854775808(16_8000000000000000),[16,10])
 ItkOpenDeclListSep(L3(16_3),[11,4])
 ItkConstDeclLtPatch(C187,[15,10])
 ItkConstDeclRt([15,10])
 ItkLiteral(96 StkIntLit,(16_1),[15,16])
 ItkConstDeclValue([15,14])
 ItkDeclTypeAbsent([15,14])
 ItkConstDeclType([15,14])
 ItkDeclId(L9(16_9),Id8"MV2",L-9223372036854775808(16_8000000000000000),[15,10])
 ItkOpenDeclListSep(L2(16_2),[11,4])
 ItkConstDeclLtPatch(C138,[14,10])
 ItkConstDeclRt([14,10])
 ItkLiteral(96 StkIntLit,(16_0),[14,16])
 ItkConstDeclValue([14,14])
 ItkDeclTypeAbsent([14,14])
 ItkConstDeclType([14,14])
 ItkDeclId(L9(16_9),Id7"MV1",L-9223372036854775808(16_8000000000000000),[14,10])
 ItkOpenDeclListSep(L1(16_1),[11,4])
 ItkVarDeclLtPatch(C78,[11,8])
 ItkVarDeclRt([11,8])
 ItkDeclValAbsent([14,2])
 ItkVarDeclValue([11,10])
 ItkReservedIdRef(Rid"INTEGER",[11,12])
 ItkVarDeclType([11,8])
 ItkVarDeclIdListLtPatch(C78,L1(16_1),[11,8])
 ItkVarDeclIdListRt(L1(16_1),[11,8])
 ItkDeclId(L10(16_a),Id6"V",L-9223372036854775808(16_8000000000000000),[11,8])
 ItkOpenScopeLt(L3(16_3))
 ItkDeclScopeLt(L3(16_3))
 ItkOpenScopeLt(L2(16_2))
 ItkDeclScopeRt(L2(16_2))
 ItkProcDefBody([11,2])
 ItkSignatureProperLtPatch(C28,[9,14])
 ItkSignatureProperRt([9,14])
 ItkRaisesSetRt(L0(16_0),[11,2])
 ItkRaisesSetLt(L0(16_0),[11,2])
 ItkResultTypeAbsent([11,2])
 ItkFormalsListLtPatch(C28,L0(16_0),[9,16])
 ItkFormalsListRt(L0(16_0),[9,16])
 ItkDeclScopeLt(L2(16_2))
 ItkDeclId(L18(16_12),Id5"P",L-9223372036854775808(16_8000000000000000),[9,12])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
